packages/core/src/components/essentials/modal.ts

```diff
@@ -23,13 +23,14 @@ export class Modal extends Component {
 	}
 
 	handleShowModal = () => {
+		const id = this.services.domUtils.getChartID()
 		this.modal
 			.attr('data-modal', true)
 			.attr('class', 'cds--modal')
 			.attr('role', 'dialog')
 			.attr('aria-modal', true)
-			.attr('aria-labelledby', 'modal-title')
-			.attr('aria-describedby', 'modal-description')
+			.attr('aria-labelledby', `${id}__modal-title`)
+			.attr('aria-describedby', `${id}__modal-description`)
```

Comment:
these just point to an ID, but you also need to set these IDs to the modal

(currently the modal IDs are set to `modal-title` & `modal-description`, so we need to add the `${id}__` in there

Reply:
over here

![image](https://github.com/carbon-design-system/carbon-charts/assets/14989804/865b6605-82c0-4645-a7bf-f7103e9c9435)